bigtable/client/table.h

```diff
@@ -56,6 +59,9 @@ inline std::string TableName(std::shared_ptr<DataClient> client,
  */
 class Table {
  public:
+  /// Signifies that `Table::ReadRows()` should not limit the number of rows.
+  static int const NO_ROWS_LIMIT = RowReader::NO_ROWS_LIMIT;
```

Comment:
We could use two overloads instead of a magic value...  Not sure what is best, I just dislike magic values.

Reply:
Zero gets special treatment in the protocol, but I agree we can protect the users from accidentally passing it. I've added an overload and enforced that rows_limit, if given, is strictly greater than 0.